packages/components/src/components/checkbox/checkbox.lite.tsx

```diff
@@ -192,6 +194,10 @@ export default function DBCheckbox(props: DBCheckboxProps) {
 					disabled={getBoolean(props.disabled, 'disabled')}
 					value={props.value}
 					required={getBoolean(props.required, 'required')}
+					{...(getBooleanAsString(props.requiredAnnotation) ===
+					'false'
+						? { 'data-required-icon': 'false' }
+						: {})}
```

Comment:
```suggestion
data-hide-required-asterisk={getHideProp(props.showRequiredAsterisk)}
```

We have already something `data-hide-icon={getHideProp(props.showIcon)}` for many components. I would like to stick with this format for every prop which is `true` at the beginning. 
Furthermore, I think we should name the prop `showRequiredAsterisk` what do you think?